### ISSUE TITLE
Is lin-vst-server32Srv.exe a virus?

### ISSUE THREAD

**Issue by lucmp** [osxmidi/LinVst#110]
I decided to upgrade linvst and had to replace those four files in /usr/bin:

- lin-vst-servertrack.exe
- lin-vst-servertrack.exe.so
- lin-vst-servertrack32.exe
- lin-vst-servertrack32.exe.so

Then I noticed this additional file:
lin-vst-server32Srv.exe

I was suspicious and ran some antivirus on it. Comodo says it's infected with MalCrypt.Indus!@105441913

I searched for that info and found that other people have detected the same problem in other files, and they tend to conclude that it's a false positive.

Maybe it is, but note that the file name is slightly different from the regular linvst files: lin-vst-server32Srv.exe

That is suspicious. Do you know anything about it?

**Comment by lucmp** [comment]
Like I said, it is a virus. The files seem to have been created many months ago, when I was testing Bitwig. I remember I had a problem with it, sound wouldn't work for regular users. No problem with other applications, it was just Bitwig. So I ran it as root, and sound worked.

I have scanned all my plugins all over again and the mysterious file didn't turn up. But I know I have deleted a ton of plugins since then, so now it's rather difficult to identify the culprit. Sorry about this. I wish I could have come up with useful information.
